pypeit/specobjs.py

```diff
@@ -222,6 +222,7 @@ def unpack_object(self, ret_flam=False, extract_type='OPT'):
         flux_key = '{}_{}'.format(extract_type, flux_attr)
         wave_key = '{}_WAVE'.format(extract_type)
         # Test
+        msgs.info(f'checking that flux_key {flux_key} is available')
```

Comment:
This information statement is not needed I think.

Reply:
removed.